PDFtoPrinter/GetPrinters.cs

```diff
@@ -0,0 +1,62 @@
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Management;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+
+namespace PDFtoPrinter
+{
+    public sealed class GetPrinters

```

Comment:
Hi @itsmuntadhar ,
Thanks for your contribution! It's just a small one. All the other classes in the project are nouns, and this one is more like a verb. Can you rename the class to something like "PrintersQuery" and the method name below "Run" or a similar concept just to keep the source code consistent?

Reply:
sure, will do it ASAP

Reply:
@svishnevsky update is live now.
hope you find time to review it again.